package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@ava/babel-preset-stage-4",
+  "version": "0.0.0",
+  "description": "Efficiently applies the minimum of transforms to run stage 4 code on Node.js 4 and 6",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "plugins"
+  ],
+  "scripts": {
+    "test": "ava",
+    "posttest": "xo",
+    "coverage": "nyc npm test"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/avajs/babel-preset-stage-4.git"
+  },
+  "keywords": [
+    "ava",
+    "babel"
+  ],
+  "author": "Mark Wubben (https://novemberborn.net/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/avajs/babel-preset-stage-4/issues"
+  },
+  "homepage": "https://github.com/avajs/babel-preset-stage-4#readme",
+  "devDependencies": {
+    "ava": "^0.17.0",
+    "coveralls": "^2.11.15",
```

Comment:
Adding coverage and coveralls for this tiny module feels a bit overkill, but I guess it's ok, since you've already done the work.